Paraswap-Agents/AdminOperations/README.md

```diff
@@ -0,0 +1,63 @@
+# Admin Operations Agent
+
+## Description
+
+This agent detects admin operations on `AugustusSwapper` Contract.
+
+## Supported Chains
+
+- Ethereum
+- Polygon
+- BSC
+- Avalanche
+- Fantom
+- Ropsten
+
+## Alerts
+
+- PARASWAP-1-1
+
+  - Fired when the `Router` or `Adapter` has been initialized.
+  - Severity is always set to "Info".
+  - Type is always set to "Info".
+  - Metadata fields:
+    - `address`: Address of the initialized router/adapter.
+
+- PARASWAP-1-2
+
+  - Fired when the tokens were transfers through calls to `transferTokens`.
+  - Severity is always set to "Info".
+  - Type is always set to "Info".
+  - Metadata fields:
+    - `token`: Address of the transfered token.
+    - `destination`: Address to which the tokens were transfered.
+    - `amount`: Amount of transfered tokens.
+
+- PARASWAP-1-3
+
+  - Fired when the router implementation was upgraded through calls to `setImplementation`.
```

Comment:
The `AugustusSwapper` smart contract supports multiple router implementations. 
It is possible to either
- Upgrade a router implementation by overwriting it's existing selector
- Add a new router implementation by using a unique selector that doesn't currently exist in the array `selectorVsRouter[]`

Adding checks to differentiate between the two would make the agent need to call to on-chain data which would slow the agent down. May be worth asking the client what they would prefer for this finding.